lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -594,6 +594,64 @@ class EmitterBase {
                                  std::function<void(Value)> operandEmitter,
                                  ArrayAttr symAttrs, ModuleNameManager &names);
 
+  /// Emit the value of a StringAttr as one or more Verilog "one-line" comments
+  /// ("//").  Break the comment to respect the emittedLineLength and trim
+  /// whitespace after a line break.  Do nothing if the StringAttr is null or
+  /// the value is empty.
+  void emitComment(StringAttr comment) {
```

Comment:
Plz move the implementation of this method out of line from the class declaration, it is pretty big.

Reply:
Good thinking! Much cleaner that way. Fixed in the rebase.